src/AspNet.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -22,22 +22,61 @@ public static class Algorithms {
         }
 
         public static class Claims {
-            public const string Active = "active";
-            public const string Address = "address";
-            public const string AtHash = "at_hash";
-            public const string ClientId = "client_id";
-            public const string Confidential = "confidential";
-            public const string EmailVerified = "email_verified";
-            public const string PhoneNumber = "phone_number";
-            public const string PhoneNumberVerified = "phone_number_verified";
-            public const string PreferredUsername = "preferred_username";
-            public const string Profile = "profile";
-            public const string Scope = "scope";
-            public const string TokenId = "token_id";
-            public const string TokenType = "token_type";
-            public const string Usage = "usage";
-            public const string Username = "username";
-            public const string Website = "website";
+            public static class Protocol {
+                public const string AtHash = "at_hash";
+                public const string Audience = "aud";
+                public const string AuthorizedParty = "azp";
+                public const string Confidential = "confidential";
+                public const string CryptographicHash = "c_hash";
```

Comment:
>FYI, there's a fixed set of strongly typed properties in the userinfo context class that you can use to set the corresponding claims. Using the claims const class shouldn't be necessary in this case.

That's fine, but what happens if someone wants to target identity or access token destinations based on the claim types in, say, a UserManager? While not required, having the constants is definitely helpful.

I also agree with @DovydasNavickas about having a bazillion options via code completion, and I tend to organize my own constants in a hierarchical pattern for organizational purposes because of this. It can be changed in the next version, sure, but since it was already done, I'm not sure why not to use it. 😕 

I think maybe you just like to modify PR code. 😆